src/automations/checkInactivity.js

```diff
@@ -44,10 +45,14 @@ async function scrapePullRequests(client, pullRequests) {
     }).map(c => c.commit.message);
     const bodRef = client.automations.get("issueReferenced").findKeywords(body);
 
-    if (bodRef || refIssues.length) {
-      const com = refIssues[0];
-      const ref = com ? com.match(/#([0-9]+)/)[1] : body.match(/#([0-9]+)/)[1];
-      references.set(`${repoName}/${ref}`, time);
+    if (bodRef && refIssues.length) {
+      const commitRef = refIssues[0].match(/#([0-9]+)/)[1];
```

Comment:
I understand that you were trying to resolve the merge conflict, but this completely reverts the previous commit, so you need to fix this.